lib/generators/paper_trail/install_generator.rb

```diff
@@ -31,6 +32,13 @@ def create_migration_file
       end
     end
 
+    def create_initializer
+      create_file(
```

Comment:
> Does it make sense to only generate this file if options.with_associations? == true since the default is false? 

Yeah, that would make sense. One fewer initializer file for most people (people not using the experimental associations features)